pgtest_test.go

```diff
@@ -33,8 +49,8 @@ func TestPersistent(t *testing.T) {
 	dir, err := ioutil.TempDir("", "pgtest")
 	assert.NoError(err)
 	defer os.RemoveAll(dir)
-
-	pg, err := pgtest.StartPersistent(dir)
+	config := pgtest.NewConfig().Persistent().DataDir(dir)
```

Comment:
Please don't change existing tests, they're a guarantee that we don't break existing code out there. Only ever add new tests.